GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -49,6 +49,25 @@ public RevisionDiffControl()
             _findFilePredicateProvider = new FindFilePredicateProvider();
             _gitRevisionTester = new GitRevisionTester(_fullPathResolver);
             _revisionDiffContextMenuController = new FileStatusListContextMenuController();
+            DiffText.TopScrollReached += DiffText_TopScrollReached;
+            DiffText.BottomScrollReached += DiffText_BottomScrollReached;
+        }
+
+        private void DiffText_TopScrollReached(object sender, EventArgs e)
+        {
+            if (Control.ModifierKeys == Keys.Alt || AppSettings.ContinuousScrollToNextFile)
+            {
+                DiffFiles.SelectPreviousVisibleItem();
+                DiffText.ScrollToBottom();
```

Comment:
Does this flicker?

Reply:
Yes, very briefly a little extra.
As the file is already updated which appears as a kind of flicker itself, it is not really noticed in itself.

You can get the entire PR to look ugly if (like it is a screen2gif) if you speed view the changes in a commit.
If you actually read one line in every file, it is a nice functionality..